Cubical/Codata/M/AsLimit/Coalg/Properties.agda

```diff
@@ -0,0 +1,207 @@
+{-# OPTIONS --cubical --guardedness #-} --safe
+
+open import Cubical.Data.Unit
+open import Cubical.Data.Sigma
+open import Cubical.Data.Nat
+open import Cubical.Data.Prod
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Function using ( _∘_ ; idfun )
+open import Cubical.Foundations.Transport
+open import Cubical.Foundations.Univalence
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.GroupoidLaws
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Path
+
+open import Cubical.Functions.Embedding
+open import Cubical.Functions.FunExtEquiv
+
+open import Cubical.Codata.M.AsLimit.helper
+
+open import Cubical.Codata.M.AsLimit.Coalg.Base
+open import Cubical.Codata.M.AsLimit.Container
+open import Cubical.Codata.M.AsLimit.M
+
+module Cubical.Codata.M.AsLimit.Coalg.Properties where
+
+open Iso
+
+-- Construction of Coalgebras from M-types, defined in
+-- https://arxiv.org/pdf/1504.02949.pdf
+-- "Non-wellfounded trees in Homotopy Type Theory"
+-- Benedikt Ahrens, Paolo Capriotti, Régis Spadotti
+
+-----------------------------------------------------------------------------
+-- The limit of a Polynomial functor over a Container is a Final Coalgebra --
+-----------------------------------------------------------------------------
+
+Ps : ∀ {ℓ} -> (S : Container ℓ) -> (C,γ : Coalg₀ S) -> Container ℓ
```

Comment:
What does Ps and Ms stand for?

Reply:
Also, they don't seem to be used anywhere?

Reply:
I have removed them.